### DISCUSSION TITLE
SLOW - is it just me or thats how it is for everyone.

### DISCUSSION THREAD

**Discussion by balajeek** [sct/overseerr#2591] [General]
I have docker version of overseer and linked with plex, sonarr, radarr
accessing the UI for any pages takes long time to render.

functionality wise no problem it works well.

**Comment by Nelinski** [comment]
Nope, fine for me.

**Reply by balajeek** [reply]
what machine/config you are running this on?

**Comment by pm-melo** [comment]
It's the same for me. I believe it started when it was updated to the current version. It takes so long to load and hangs, causing Uptime Kuma to timeout and say it's unavailable.

**Comment by pm-melo** [comment]
Hey @balajeek! 
Just chiming in to tell you that I was tinkering, rolling back versions of the docker image to try to see where it got slower and turns out in my case apparently what made it slower is the use of CSRF Protection. Once I turned it off and rebooted the container everything was much snapier. You'll find this setting under the General tab. Hope it helps somehow

**Reply by balajeek** [reply]
thanks for the try, I looked in my settings and its already unchecked.  so i guess that may not be the reason.

**Reply by pm-melo** [reply]
You're right, my instance is not up to speed again. I installed a fresh instance on a free cloud provider using the same setup I have locally and speed is great. I'm wondering if it's possible to have something to do with sonarr and radarr's configurations (this new instance doesn't have that configured), but I doubt it.
Imo, it feels like either my requests to external APIs are being slowed down (maybe IP related?) or the data directory is somehow for this service being hammered. Thinking about setting up from scratch locally but afraid it won't be the fix I need.

**Reply by ertechdesign** [reply]
Disabling CSFR Protection made the trick for me, thank you very much for this I spent hours trying to fix it. It's still not very fast (any searches and images load in about 6-10 seconds but before it was more than 30 seconds so it was a total pain to use) but it's much better now

**Comment by eximo84** [comment]
Very slow for me. Always has been. It just looks so much nicer than ombi.
